Community/ideAlarm/automation/lib/python/community/idealarm/__init__.py

```diff
@@ -166,6 +166,7 @@ def __init__(self, parent, zoneNumber, cfg):
         self.name = cfg['name']
         self.armAwayToggleSwitch = cfg['armAwayToggleSwitch']
         self.armHomeToggleSwitch = cfg['armHomeToggleSwitch']
+        self.autoResetAfterAlert = cfg['autoReset']
```

Comment:
Since this is a new configuration key, I'd prefer if we check for the existance of the key instead of assuming it's already in the configuration. (E.g. if key in dict: ) I'd prefer if this new feature won't be a "breaking change", that is, if configuration key is not added, things will just keep working as they did before.

Reply:
Is there a default value, so that you could use something like this?

    self.autoResetAfterAlert = cfg.get('autoReset', default_value)

Reply:
I have implemented it with a check for the key. Your suggestion seems a little bit more compact, but the result will be the same?

Reply:
Exactly the same